### PR TITLE
gen3staging_kf : deploying 2020.08 and WAF to commons

### PR DESCRIPTION
**Pull request by atharvar28**

### CODE REVIEW

File: gen3staging.kidsfirstdrc.org/manifest.json [hunk 1]
```diff
@@ -7,15 +7,15 @@
...
-    "dashboard": "quay.io/cdis/gen3-statics:1.0.0",
-    "fence": "quay.io/cdis/fence:2020.05",
-    "indexd": "quay.io/cdis/indexd:2020.05",
-    "peregrine": "quay.io/cdis/peregrine:2020.05",
-    "pidgin": "quay.io/cdis/pidgin:2020.05",
-    "revproxy": "quay.io/cdis/nginx:2020.05",
-    "sheepdog": "quay.io/cdis/sheepdog:2020.05",
-    "portal": "quay.io/cdis/data-portal:2020.05",
+    "arborist": "quay.io/cdis/arborist:2020.08",
+    "dashboard": "quay.io/cdis/gen3-statics:2020.,08",
```
Comment (paulineribeyre) [inline gen3staging.kidsfirstdrc.org/manifest.json:11]:
```suggestion
    "dashboard": "quay.io/cdis/gen3-statics:2020.08",
```

**Comment by PlanXCyborg** [comment]
# gen3staging.kidsfirstdrc.org/manifest.json
## :warning: Services on branch
- dashboard
## Deployment changes
- indexd
  - Indexd's blank record endpoints now allow direct iteration from a user with their access token but requires the file upload arborist role to be modified so the service is not just `fence` but instead `*` so indexd works too. see https://github.com/uc-cdis/commons-users/pull/908/files for example (uc-cdis/uc-cdis/indexd#275)
- portal
  - Portal sould be updated to include this fix if `Indexd` is above `2.12.0` or `2020.07` and user want to use Arborist UI (uc-cdis/uc-cdis/data-portal#702)
  - New optional config field in the index section of gitops.json called homepageChartNodeChunkSize. Sets the value for paritioning requests to Peregrine for the homepage chart. Defaults to 15 if not provided. (uc-cdis/uc-cdis/data-portal#699)
  - New `explorerConfig` which can take configs for multiple tabs, backward compatiable with current `dataExplorerConfig` and `fileExplorerConfig` (uc-cdis/uc-cdis/data-portal#676)

File: gen3staging.kidsfirstdrc.org/manifest.json (hunk 1 above)
Comment (atharvar28) [reply gen3staging.kidsfirstdrc.org/manifest.json]: 🤦  I will fix it

**Comment by PlanXCyborg** [comment]
# gen3staging.kidsfirstdrc.org/manifest.json
## Deployment changes
- indexd
  - Indexd's blank record endpoints now allow direct iteration from a user with their access token but requires the file upload arborist role to be modified so the service is not just `fence` but instead `*` so indexd works too. see https://github.com/uc-cdis/commons-users/pull/908/files for example (uc-cdis/uc-cdis/indexd#275)
- portal
  - Portal sould be updated to include this fix if `Indexd` is above `2.12.0` or `2020.07` and user want to use Arborist UI (uc-cdis/uc-cdis/data-portal#702)
  - New optional config field in the index section of gitops.json called homepageChartNodeChunkSize. Sets the value for paritioning requests to Peregrine for the homepage chart. Defaults to 15 if not provided. (uc-cdis/uc-cdis/data-portal#699)
  - New `explorerConfig` which can take configs for multiple tabs, backward compatiable with current `dataExplorerConfig` and `fileExplorerConfig` (uc-cdis/uc-cdis/data-portal#676)

**Comment by PlanXCyborg** [comment]
# gen3staging.kidsfirstdrc.org/manifest.json
## Deployment changes
- indexd
  - Indexd's blank record endpoints now allow direct iteration from a user with their access token but requires the file upload arborist role to be modified so the service is not just `fence` but instead `*` so indexd works too. see https://github.com/uc-cdis/commons-users/pull/908/files for example (uc-cdis/uc-cdis/indexd#275)
- portal
  - Portal should be updated to include this fix if `Indexd` is above `2.12.0` or `2020.07` and user want to use Arborist UI (uc-cdis/uc-cdis/data-portal#702)
  - New optional config field in the index section of gitops.json called homepageChartNodeChunkSize. Sets the value for paritioning requests to Peregrine for the homepage chart. Defaults to 15 if not provided. (uc-cdis/uc-cdis/data-portal#699)
  - New `explorerConfig` which can take configs for multiple tabs, backward compatiable with current `dataExplorerConfig` and `fileExplorerConfig` (uc-cdis/uc-cdis/data-portal#676)

**Comment by paulineribeyre** [comment]
- fence
  - If the fence config already has [CLIENT_, USER_, SESSION_]ALLOWED_SCOPES fields, and GA4GH AAI integration is desired, then add ga4gh_passport_v1 to all those lists. If default config values are being used then no change required. (uc-cdis/uc-cdis/fence#799)

**Pull request merged**